### PR TITLE
Release/extensions

### PR DESCRIPTION
**Pull request by rrgiles**
Added extension #29 for Idera SQL DM Performance Insights extension in extensionGallery.json and extensionGallery-insider.json files.

### CODE REVIEW

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -1023,6 +1023,63 @@
...
+								}
+							],
+							"properties": [
+								{
+									"key": "Microsoft.VisualStudio.Code.ExtensionDependencies",
+									"value": ""
+								},
+								{
+									"key": "Microsoft.VisualStudio.Code.Engine",
+									"value": *"
```
Comment (alanrenmsft) [inline extensionsGallery-insider.json]: missing a quote

File: extensionsGallery-insider.json [hunk 2]
```diff
@@ -1023,6 +1023,63 @@
...
+									"value": "https://www.idera.com/"
+								}
+							]
+						}
+					],
+					"statistics": [],
+					"flags": "preview"
 				}
 			],
 			"resultMetadata": [
```
Comment (alanrenmsft) [inline extensionsGallery-insider.json]: please bump up the count number as you have added a new one in this PR.

File: extensionsGallery.json [hunk 3]
```diff
@@ -1023,6 +1023,63 @@
...
+								}
+							],
+							"properties": [
+								{
+									"key": "Microsoft.VisualStudio.Code.ExtensionDependencies",
+									"value": ""
+								},
+								{
+									"key": "Microsoft.VisualStudio.Code.Engine",
+									"value": *"
```
Comment (alanrenmsft) [inline extensionsGallery.json]: missing quote

File: extensionsGallery.json [hunk 4]
```diff
@@ -1023,6 +1023,63 @@
...
+									"value": "https://www.idera.com/"
+								}
+							]
+						}
+					],
+					"statistics": [],
+					"flags": "preview"
 				}
 			],
 			"resultMetadata": [
```
Comment (alanrenmsft) [inline extensionsGallery.json]: same here, bump up the count please.

File: extensionsGallery-insider.json (hunk 1 above)
Comment (rrgiles) [reply extensionsGallery-insider.json]: added

File: extensionsGallery-insider.json (hunk 2 above)
Comment (rrgiles) [reply extensionsGallery-insider.json]: count bumped to 19

File: extensionsGallery.json (hunk 3 above)
Comment (rrgiles) [reply extensionsGallery.json]: added

File: extensionsGallery.json (hunk 4 above)
Comment (rrgiles) [reply extensionsGallery.json]: count bumped to 19

**Pull request merged**